CTestConfig.cmake

```diff
@@ -0,0 +1,18 @@
+## This file should be placed in the root directory of your project.
+## Then modify the CMakeLists.txt file in the root directory of your
+## project to incorporate the testing dashboard.
+##
+## # The following are required to submit to the CDash dashboard:
+##   ENABLE_TESTING()
+##   INCLUDE(CTest)
+
+set(CTEST_PROJECT_NAME WarpX)
+set(CTEST_NIGHTLY_START_TIME 08:00:00 UTC)
+
+set(CTEST_SUBMIT_URL https://my.cdash.org/submit.php?project=WarpX)
+
+set(CTEST_DROP_SITE_CDASH TRUE)
+
+# Additional settings
+set(CTEST_SITE "Azure-Pipelines")
+set(CTEST_BUILD_NAME "CI-Development")
```

Comment:
I wonder if these two should be set or if one can pass them to CMake/CTest... In the end, we will run the same file from Perlmutter and other sites, where the name and site does not really fit.

Or does one approach this with multiple config files?